src/main/java/org/mifos/connector/ams/zeebe/ZeebeeWorkers.java

```diff
@@ -589,6 +589,29 @@ public void setupWorkers() {
                         .name(WORKER_ACCOUNT_IDENTIFIER + dfspid)
                         .maxJobsActive(workerMaxJobs)
                         .open();
+
+                logger.info("## generating " + WORKER_SEND_CALLBACK + " worker");
```

Comment:
donot use + to concatenate logs.
use {}

Reply:
done